data/en/mbs_0161.json

```diff
@@ -0,0 +1,380 @@
+                        "id": "introduction",
+                        "primary_content": [{
+                            "type": "Basic",
+                            "id": "use-of-information",
+                            "content": [{
+                                "list": [
+                                    "Data should relate to all sites in England, Scotland, Wales and Northern Ireland.",
+                                    "You can provide informed estimates if actual figures aren’t available.",
+                                    "We will treat your data securely and confidentially."
+                                ]
+                            }]
+                        }],
+                        "preview_content": {
+                            "id": "preview",
+                            "title": "Information you need",
+                            "content": [{
+                                "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
+                            }],
+                            "questions": [{
+                                    "question": "Total Turnover",
+                                    "content": [{
+                                            "description": "Include:",
+                                            "list": [
+                                                "exports",
+                                                "payments for work in progress",
+                                                "costs incurred and passed on to customers",
+                                                "income from sub-contracted activities",
+                                                "commission",
+                                                "sales of goods purchased for resale",
+                                                "revenue earned from other parts of the business not named (please supply at fair value)"
+                                            ]
+                                        },
+                                        {
+                                            "description": "Exclude:",
+                                            "list": [
+                                                "VAT",
+                                                "excise duties",
+                                                "income from the sale of fixed capital assets",
+                                                "grants and subsidies",
+                                                "insurance claims",
+                                                "interest received"
+                                            ]
+                                        }
+                                    ]
+                                },
+                                {
+                                    "question": "Changes in total turnover",
```

Comment:
Your missing also 'Number of employees' and 'Employees and working hours' here include/excludes here.

Reply:
Very weird, It seems I deleted it just before raising the PR.